components/core/tools/scripts/lib_install/liblzma.sh

```diff
@@ -60,7 +75,30 @@ mkdir build
 cd build
 cmake -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE ../
 make -j${num_cpus}
-make install liblzma
+
+# Check if checkinstall is installed
+set +e
+command -v checkinstall
+checkinstall_installed=$?
+set -e
+
+# Install
+if [ $checkinstall_installed -eq 0 ] ; then
+  install_cmd_args+=(
+    checkinstall
+    --default
+    --fstrans=no
+    --nodoc
+    --pkgname "${package_name}"
+    --pkgversion "${version}"
+    --provides "${package_name}"
+    --pakdir "${deb_output_dir}"
+  )
+fi
+install_cmd_args+=(
+    make install liblzma
```

Comment:
```suggestion
    make install
```

Reply:
Thanks for the update. Also, I'm pretty sure the `liblzma` at the end here is superfluous.
Otherwise, I don't have further comments unless someone asks me to review the details of the bash scripts